cluster-autoscaler/config/crd/provisioning.k8s.io_provisioningrequests.yaml

```diff
@@ -0,0 +1,203 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    api-approved.kubernetes.io: https://github.com/kubernetes/autoscaler/pull/5848
```

Comment:
commented at https://github.com/kubernetes/autoscaler/pull/5848#issuecomment-1719515802 but wanted to signal boost here... that PR didn't get API approval yet – we're reviewing the updates today.

If you want to merge ahead of API approval, you can work in the `autoscaling.x-k8s.io` API group with only SIG oversight, which explicitly marks the API as experimental and doesn't require API approval.